Tests/XCUITests/IntegrationTests.swift

```diff
@@ -94,12 +94,12 @@ class IntegrationTests: BaseTestCase {
         navigator.goto(BrowserTabMenu)
         navigator.goto(Intro_FxASignin)
         navigator.performAction(Action.OpenEmailToSignIn)
-        waitForExistence(app.navigationBars[AccessibilityIdentifiers.Settings.FirefoxAccount.fxaNavigationBar], timeout: TIMEOUT_LONG)
-        waitForExistence(app.webViews.textFields[AccessibilityIdentifiers.Settings.FirefoxAccount.emailTextFieldChinaFxA], timeout: TIMEOUT_LONG)
 
-        // Wait for element not present on FxA sign in page China FxA server
-        waitForNoExistence(app.webViews.otherElements.staticTexts["Firefox Monitor"])
-        XCTAssertFalse(app.webViews.otherElements.staticTexts["Firefox Monitor"].exists)
+        sleep(5)
```

Comment:
I think 1waitForExistence` should be good enough - I wouldn't sleep the thread unless you really have to. If you do, a quick comment for why would be helpful.